deployment/hasura/metadata/databases/tables/actions/action_run.yaml

```diff
@@ -0,0 +1,47 @@
+table:
+  name: action_run
+  schema: actions
+configuration:
+  custom_name: "action_run"
+object_relationships:
+  - name: action_definition
```

Comment:
Nonblocking: rename suggestion 
```suggestion
  - name: action
```

Reply:
Bumping as you left a 👍 but nothing was changed. Did you mean to commit a change or were you rejecting the suggestion?